src/me/ryanhamshire/GriefPrevention/GriefPrevention.java

```diff
@@ -256,6 +256,17 @@ public void onEnable()
 		this.customLogger = new CustomLogger();
         
 		AddLogEntry("Finished loading configuration.");
+
+                // Check we can run with this Minecraft version
+                if (!Bukkit.getVersion().startsWith("1.12")) {
```

Comment:
I'd like getServer() but there's already a lot of static usage in this I guess... Also, the current and upcoming version is compatible with 1.11.2 as well.

Might need to mark this with a `TODO` so we remember to change it or something